src/pages/EventFeedbackAnalytics.tsx

```
@@ -0,0 +1,432 @@
+];
+
+const CATEGORIES = ['All', 'Technology', 'Cultural', 'Career', 'Social'];
+const SENTIMENT_COLORS = { positive: '#22c55e', neutral: '#3b82f6', negative: '#ef4444' };
+
+/* ─────────────────────── SVG COMPONENTS ─────────────────────── */
+function RatingRing({ value, max = 5, size = 80 }) {
+  const pct = (value / max) * 100;
+  const radius = (size - 10) / 2;
+  const circumference = 2 * Math.PI * radius;
+  const offset = circumference - (pct / 100) * circumference;
+  const color = value >= 4.5 ? '#22c55e' : value >= 3.5 ? '#3b82f6' : value >= 2.5 ? '#f59e0b' : '#ef4444';
```

ESLint

[error] Replace `·value·>=·4.5·?·'`#22c55e`'·:·value·>=·3.5·?·'`#3b82f6`'·:·value·>=·2.5·?·'`#f59e0b`'·:·'`#ef4444`'` with `⏎····value·>=·4.5·?·"`#22c55e`"·:·value·>=·3.5·?·"`#3b82f6`"·:·value·>=·2.5·?·"`#f59e0b`"·:·"`#ef4444`"`

(prettier/prettier)